### PR TITLE
Resolve a number of accessibility issues

### PR DESCRIPTION
**Pull request by bazzadp**
Part of #250 

Resolves a number of accessibility issues raised by Axe, WAVE and Firefox accessibility tab

### CODE REVIEW

File: config/reports.json [hunk 1]
```diff
@@ -1038,7 +1038,7 @@
     }
   },
   "chrome-ux-report": {
-    "name": "Chrome User Experience Report",
+    "name": "Chrome User Experience",
```
Comment (rviscomi) [inline config/reports.json:1041]: "Report" is part of the product name

**Review by rviscomi — CHANGES_REQUESTED** [review]

File: config/reports.json (hunk 1 above)
Comment (bazzadp) [reply config/reports.json:1041]: The reason I removed it is cause some of the a11y tools were (not unreasonably) complaining about the confusion of two "Chrome User Experience Report" links, linking to two difference places:

image

The heading links to the HTTP Archive CrUX Report (The Chrome User Experience Report Report if you will!), while the second link goes to the CrUX home page.

It's a minor thing so can just revert and ignore it, unless you've any better suggestions on how to handle this?

Comment (bazzadp) [reply config/reports.json:1041]: There's also the fact I renamed the buttons so we wouldn't have all the buttons called "View the Report", but a more descriptive button:

image

If we revert "Chrome User Experience" back to "Chrome User Experience Report", then it's button will become "View the Chrome User Experience Report Report" which, while technically accurate is a bit of a mouthful!

Plus I didn't like the fact the original title wrapped when none of the others did 😁

Comment (rviscomi) [reply config/reports.json:1041]: How about renaming the report to `CrUX`? Shorter and allows "Report" after.

Comment (bazzadp) [reply config/reports.json]: Done!

**Pull request merged**